src/SideAppbar/SideAppbar.js

```diff
@@ -0,0 +1,130 @@
+import { Box, Link, Paper, Typography } from "@mui/material";
+
+import AppLogo from "./AppLogo";
+import PropTypes from "prop-types";
+import React from "react";
+
+// SideAppbar component for app which displays logo, list of items and app version
+function SideAppbar({ logoLinkUrl = null, showLogo = true, onButtonClick }) {
```

Comment:
It took me ages to find this file because of the naming. I think maybe AppLauncher would be a better name?

Reply:
Renamed